.cypress/cypress/integration/merton.js

```diff
@@ -0,0 +1,24 @@
+it('loads the Merton FMS Pro front page', function() {
+    cy.visit('http://merton.localhost:3001/');
+    cy.contains('Merton Council');
+});
+it('does not allow typing >255 chars in additional question fields', function() {
+    cy.server();
+    cy.visit('http://merton.localhost:3001/report/new?latitude=51.40097&longitude=-0.19655');
+    cy.get('input[value="Flytipping"]').click();
+    cy.get('button').first().click();
+    var longstring260 = 'a'.repeat(260);
+    cy.get('input#form_evidence').as('evidence');
+    cy.get('@evidence').type('a'.repeat(260));
+    cy.get('@evidence').invoke('val').then(function(val){
+        expect(val.length).to.be.at.most(255);
+    });
+});
+it('does not allow user to continue if additional question field value is >255', function() {
+    cy.get('@evidence').invoke('val', longstring260);
```

Comment:
This is failing because `longstring260` isn't in scope here.

Reply:
Thanks Dave, sorry, I usually do do this, have added another reminder! 🤦🏼‍♂️